index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@300&display=swap'); </style>
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@300&display=swap'); </style>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="mobile-view">
+            <header class="mobile-header">
+                <div class="display-header">
+                    <p id="time">9:41</p>
+                    <p id="portfolio-name">John Doe </p>
+                </div>
+                <div class="signal-div">
+                    <div class="signal-div-items">
+                        <img src="images/Cellular Connection.png" alt="cellular connection image">
+                        <img src="images/Wifi.png" alt="wifi image">
+                        <img src="images/Battery.png" alt="Batteryimage">
+                    </div>
+                    <div class="union-div">
+                        <img src="images/Union.png" alt="hamburger">
+                    </div>
+                </div>
```

Comment:
- Hey, great job with the header section, however, let's wrap the content in it in a nav tag to enforce the use of html5 semantic tag.
- The content in the header should only be your name and the hamburger. so let's please remove the rest content added as the image was taken from a mobile phone that's why you see them there.

Reply:
Thank you @zieeco . All changes have been made.

Reply:
@zieeco The dashboard says changes required, but I have not received a feedback after I made the changes you required